From: Michael Olbrich <m.olbrich@pengutronix.de>
Subject: remove wrong include path.

Don't use $(prefix)/include as include path. It can break things
when cross-compiling with DESTDIR and prefix=/usr

Signed-off-by: Michael Olbrich <m.olbrich@pengutronix.de>
---
 configure.ac |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: xf86-video-vesa-2.2.1/configure.ac
===================================================================
--- xf86-video-vesa-2.2.1.orig/configure.ac
+++ xf86-video-vesa-2.2.1/configure.ac
@@ -89,7 +89,7 @@ fi
 AM_CONDITIONAL(XSERVER_LIBPCIACCESS, test "x$XSERVER_LIBPCIACCESS" = xyes)
 
 CFLAGS="$CFLAGS $XORG_CFLAGS "' -I$(top_srcdir)/src'
-INCLUDES="$XORG_INCS -I${sdkdir} "'-I$(top_srcdir)/src -I$(prefix)/include'
+INCLUDES="$XORG_INCS -I${sdkdir} "'-I$(top_srcdir)/src'
 AC_SUBST([CFLAGS])
 AC_SUBST([INCLUDES])
 
